packages/nns/src/types/governance_converters.ts

```diff
@@ -30,7 +30,7 @@ export type Action =
   | { SetDefaultFollowees: SetDefaultFollowees }
   | { Motion: Motion }
   | { SetSnsTokenSwapOpenTimeWindow: SetSnsTokenSwapOpenTimeWindow }
-  | { OpenSnsTokenSwap: OpenSnsTokenSwap };
+  | { OpenSnsDecentralizationSale: OpenSnsDecentralizationSale };
```

Comment:
so they renamed `OpenSnsTokenSwap` but, they did not rename `SetSnsTokenSwapOpenTimeWindow`?

Reply:
I believe `SetSnsTokenSwapOpenTimeWindow` might be removed. Don't they want to change how the proposal starts and ends and so on?